docs/specs/toc.yml

```diff
@@ -1895,3 +1890,52 @@ outputs:
     {"metadata": {"universal_conceptual_toc": "dotnet-test/learn/toc.json"}}
   test/a/_splitted/b/toc.json: |
     {"metadata": {"universal_conceptual_toc": "dotnet-test/learn/toc.json"}}
+---
+# Service Page href fix
```

Comment:
What is different from the existing test?

Reply:
Add description